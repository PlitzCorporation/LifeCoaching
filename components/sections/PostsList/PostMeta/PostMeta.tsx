import React from 'react';

type Props = {
	fullname: string;
	createdAt: string;
};

const PostMeta = (props: Props) => {
	const { fullname, createdAt } = props;
	return (
		<small className="text-plitz-primary mb-7 lg:mb-10">
			By {fullname} |{' '}
			{new Date(createdAt).toLocaleDateString('en-US', {
				month: 'long',
				day: '2-digit',
				year: 'numeric',
			})}
		</small>
	);
};

export default PostMeta;
