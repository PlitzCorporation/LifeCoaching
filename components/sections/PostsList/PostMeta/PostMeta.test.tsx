import { render } from '@testing-library/react';
import PostMeta from './PostMeta';

describe('PostMeta', () => {
	it('should render the component', () => {
		const { container } = render(
			<PostMeta
				fullname="Bruce Wayne"
				createdAt="1982-07-07T17:38:34.576+00:00"
			/>,
		);
		expect(container).toMatchSnapshot();
	});
});
