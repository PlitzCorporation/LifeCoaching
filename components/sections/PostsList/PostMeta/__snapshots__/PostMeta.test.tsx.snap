// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PostMeta should render the component 1`] = `
<div>
  <small
    class="text-plitz-primary mb-7 lg:mb-10"
  >
    By 
    Bruce Wayne
     |
     
    July 07, 1982
  </small>
</div>
`;
